.github/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+## Change Summary
```

Comment:
This appears to be a duplicate of `PULL_REQUEST_TEMPLATE\pull_request_template.md`

Reply:
I am investigating this one.

Reply:
I have removed the copy under the folder. I tested both options, and default template _must_ reside outside the folder, but optional templates must reside _inside_ the folder. The github docs are not completely clear on this.